modules/oe_content_tender/tests/src/Kernel/TenderKernelTestBase.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_tender\Kernel;
+
+use Drupal\Tests\rdf_entity\Kernel\RdfKernelTestBase;
+
+/**
+ * Base test class for tender content type kernel tests.
+ */
+abstract class TenderKernelTestBase extends RdfKernelTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'field',
+    'field_group',
+    'datetime_range',
+    'entity_reference_revisions',
+    'link',
+    'image',
+    'inline_entity_form',
```

Comment:
there are some modules here which are not needed, like inline_entity_form, lets remove them one by one to see which are actually needed, the less is loaded the better